src/main/java/az/gdg/mscomplaint/controller/ComplaintController.java

```diff
@@ -0,0 +1,45 @@
+package az.gdg.mscomplaint.controller;
+
+import az.gdg.mscomplaint.model.ComplaintRequest;
+import az.gdg.mscomplaint.model.dto.ComplaintDTO;
+import az.gdg.mscomplaint.service.ComplaintService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Do not use wild card in an import statement. Import all of the classes separately